ui/styles/base.css

```diff
@@ -1,29 +1,156 @@
-header {
-  margin-top: 20px;
+/* Header */
+.green {
+  color: #77AA32;
 }
 
-header li {
-  margin-left: 20px;
+.red {
```

Comment:
What's with these class names? `red` is a value, not an abstraction. `error` is an abstraction and can have other values besides `red`.